client/modules/IDE/pages/IDEView.jsx

```diff
@@ -260,12 +261,11 @@ class IDEView extends React.Component {
         <WarnIfUnsavedChanges currentLocation={this.props.location} />
         <Toast />
         <CmControllerContext.Provider value={{ current: this.cmController }}>
-          <Nav />
+          <Header
+            syncFileContent={this.syncFileContent}
+            key={this.props.project.id}
```

Comment:
I wonder if this `key` prop actually does anything?  I see it’s part of the existing code.  It doesn’t hurt to leave it but I strongly suspect that it’s useless.

Reply:
should I remove it?